tests/integration/targets/dcnm_vrf/tests/dcnm/self-contained-tests/vrf_lite.yaml

```diff
@@ -0,0 +1,286 @@
+##############################################
+##                 SETUP                    ##
+##############################################
+
+- set_fact:
+    rest_path: "/rest/control/fabrics/{{ test_fabric }}"
+  when: controller_version == "11"
+
+- set_fact:
+    rest_path: "/appcenter/cisco/ndfc/api/v1/lan-fabric/rest/control/fabrics/{{ test_fabric }}"
+  when: controller_version >= "12"
+
+- name: MERGED - Verify if fabric is deployed.
+  cisco.dcnm.dcnm_rest:
+    method: GET
+    path: "{{ rest_path }}"
+  register: result
+
+- assert:
+    that:
+    - 'result.response.DATA != None'
+
+- name: MERGED - Clean up any existing vrfs
+  cisco.dcnm.dcnm_vrf:
+    fabric: "{{ test_fabric }}"
+    state: deleted
+
+- name: VRF LITE - sleep for 40 seconds for DCNM to completely update the state
+  wait_for:
+    timeout: 40
+
+###############################################
+###                TESTS                     ##
+###############################################
+
+- name: VRF LITE- Create, Attach and Deploy new VRF - VLAN/VRF LITE EXTENSION Provided by the User in one switch
+  cisco.dcnm.dcnm_vrf: &conf1
+    fabric: "{{ test_fabric }}"
+    state: merged
+    config:
+    - vrf_name: ansible-vrf-int1
+      vrf_id: 9008011
+      vrf_template: Default_VRF_Universal
+      vrf_extension_template: Default_VRF_Extension_Universal
+      vlan_id: 500
+      source: null
```

Comment:
Do we ever set `source` or `service_vrf_template` properties?

Reply:
No we don't. I have removed both these parameter from all the plays